src/js/routes/TwitterHandleLanding.jsx

```diff
@@ -55,25 +55,26 @@ export default class TwitterHandleLanding extends Component {
 
   _onTwitterStoreChange () {
     // console.log("TwitterHandleLanding _onTwitterStoreChange");
-    let { twitter_followers_count } = TwitterStore.get();
+    let { twitter_followers_count: twitterFollowersCount } = TwitterStore.get();
```

Comment:
I had to map `twitter_followers_count` from TwitterStore to the local camel case twitterFollowersCount.